client/actions/form.js

```diff
@@ -86,33 +86,20 @@ export async function cancelAppointment(email) {
   revalidatePath("/my-appointments");
 }
 
-export async function updateVisitedOnServer(id) {
-  const response = await fetch(serverUrl + `appointments/${id}`, {
+export async function updateStatusOnServer(id, newStatus) {
```

Comment:
```suggestion
export async function updateStatus(id, newStatus) {
```
I think it's implied it's on the server since it's in this file